### DISCUSSION TITLE
Received this error message after Jedi Survivor crashed, not sure what means.

### DISCUSSION THREAD

**Discussion by broslife** [BartoszCichecki/LenovoLegionToolkit#1373] [Q&A]
I am having some issues while running Star Wars: Jedi Survivor. Sometime the game freezes and crash to desktop, sometime make the whole system freezes and the only solutions is a forced shutdown. 

Last time the game froze but crashed to desktop and then I got multiple times this message. 
<img width="401" alt="Snipaste_2024-07-19_19-36-33" src="https://github.com/user-attachments/assets/dac0c1ea-53a2-49d9-91a4-9aadf10805d3">

The screen resolution got messed up and the video drivers (Nvidia and Intel) stopped working properly. Not sure what means and not sure if the issues with game is related to Lenovo Legion Toolkit.

**Comment by BartoszCichecki** [comment]
That's a result of display driver crash. Nothing special.
